docs/cli-reference/aspire-run.md

```diff
@@ -0,0 +1,86 @@
+---
+title: aspire run command
+description: Learn about the aspire run command and its usage. This command runs an Aspire app host.
+ms.date: 07/11/2025
+---
+# aspire run command
+
+**This article applies to:** ✔️ Aspire CLI 9.4.0 and later versions.
+
+## Name
+
+`aspire run` - Run an Aspire app host in development mode.
+
+## Synopsis
+
+```dotnetcli
+aspire run [options] [[--] <additional arguments>...]
+```
+
+## Description
+
+The `aspire run` command runs the AppHost project in development mode, which configures the Aspire environment, builds and starts resources defined by the app host, launches the web dashboard, and prints a list of endpoints.
+
+[!INCLUDE [project-search-logic-description](includes/project-search-logic-description.md)]
+
+The command performs the following steps to run an apphost project:
+
+- Creates or modifies the `.aspire/settings.json` config file in the current directory, and sets the `appHostPath` config value to the path of the AppHost project file.
+- Installs or verifies that Aspire's local hosting certificates are installed and trusted.
+- Builds the AppHost project.
+- Starts the app host and its resources.
+- Starts the dashboard.
+
+The following snippet is an example of the output displayed by the `aspire run` command:
+
+```Aspire CLI
+Dashboard:  https://localhost:17178/login?t=17f974bf68e390b0d4548af8d7e38b65                                         
+            https://literate-eureka-55x5r74pwxv2vx7p-17178.app.github.dev/login?t=17f974bf68e390b0d4548af8d7e38b65   
```

Comment:
I think this is only visible when running from a DevBox or Dev Container. We might consider removing it.